packages/synapse-react-client/src/components/EntityBadgeIcons/EntityBadgeIcons.tsx

```diff
@@ -104,6 +105,11 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
 
   const { ref, inView } = useInView()
 
+  const { data: entityData } = useGetJson(entityId, versionNumber, true, {
+    enabled: inView,
+    staleTime: 60 * 1000, // 60 seconds
```

Comment:
Consider removing the `staleTime` override here and in the call to `useGetEntityBundle`. I don't think these are serving any purpose
```suggestion
```